locator/tablets.cc

```diff
@@ -100,6 +100,11 @@ tablet_migration_streaming_info get_migration_streaming_info(const locator::topo
 tablet_migration_streaming_info get_migration_streaming_info(const locator::topology& topo, const tablet_info& tinfo, const tablet_transition_info& trinfo) {
     tablet_migration_streaming_info result;
     switch (trinfo.transition) {
+        case tablet_transition_kind::rf_change:
+            if (!trinfo.pending_replica) {
+                return result;
+            }
+            [[fallthrough]];
```

Comment:
it looks like this hunk belongs to the patch that makes pending_replica optional

Reply:
Fixed.